packages/node/src/channel.ts

```diff
@@ -0,0 +1,21 @@
+import * as cf from "@counterfactual/cf.js";
+
+// Not strictly needed, added for easier legibility
+export type MultisigAddress = string;
+
+export class StateChannelInfo implements cf.legacy.channel.StateChannelInfo {
```

Comment:
do we really need this `cf.legacy.channel.StateChannelInfo`

Reply:
Seconded. Legacy will be phased out soon, and CFjs.2 won't have a concept of State Channel (might change when metachannels are implemented). Just create your own StateChannelInfo without implementing the legacy interface